src/components/Editor/EmptyLines.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { Component } from "react";
+
+import actions from "../../actions";
+import { getSelectedSource } from "../../selectors";
+
+import getEmptyLines from "../../utils/parser/getEmptyLines";
```

Comment:
we should add `getEmptyLines` as a worker task so that the work is done off the main thread

https://github.com/devtools-html/debugger.html/blob/master/docs/local-development.md#adding-a-task